cou/apps/auxiliary_subordinate.py

```diff
@@ -43,6 +43,19 @@ def current_os_release(self) -> OpenStackRelease:
 class OvnSubordinate(AuxiliarySubordinateApplication):
     """Ovn subordinate application class."""
 
+    def upgrade_plan_sanity_checks(
+        self, target: OpenStackRelease, units: Optional[list[COUUnit]]
+    ) -> None:
+        """Run sanity checks before generating upgrade plan.
+
+        :param target: OpenStack release as target to upgrade.
+        :type target: OpenStackRelease
+        :param units: Units to generate upgrade plan, defaults to None
+        :type units: Optional[list[COUUnit]], optional
+        """
+        check_ovn_version_pinning(self.name, self.config["enable-version-pinning"].get("value"))
```

Comment:
Should this check be after `upgrade_plan_sanity_checks`, meaning that first we run the default one?

Reply:
They are, but since they are raising an exception, I think it would be good to stick with the order.